extensions/ql-vscode/src/view/remote-queries/RemoteQueries.tsx

```diff
@@ -377,7 +377,7 @@ const AnalysesResults = ({
 
       <ul className="vscode-codeql__flat-list">
         {analysesResults
-          .filter(a => a.interpretedResults.length > 0 || a.rawResults)
+          .filter(a => a.interpretedResults.length > 0 || (a.rawResults?.resultSet.rows && a.rawResults.resultSet.rows.length > 0))
```

Comment:
```suggestion
          .filter(a => a.interpretedResults.length || a.rawResults?.resultSet?.rows?.length)
```
If we're willing to use the fact that `0` and `undefined` are both falsy then I think this is equivalent and a bit shorter. I have no preference whether we do this or keep it as it as though. What's there already is perfectly fine.